src/index.js

```diff
@@ -1,5 +1,7 @@
-import DotplotGL from './DotplotGL';
-import RectplotGL from './RectplotGL';
-import TickplotGL from './TickplotGL';
+import DotplotGL from "./DotplotGL";
+import RectplotGL from "./RectplotGL";
+import TickplotGL from "./TickplotGL";
 
-export {DotplotGL, RectplotGL, TickplotGL};
\ No newline at end of file
+import ReactDotplot from "./react/ReactDotPlot";
```

Comment:
may be not expose the react layer in `index.js`. the react directory should contain `index.js` with the exports so that folks can do

```
import {DotPlot} from "epiviz.heatmal.gl/react";
```